Rubberduck.Core/UI/UnitTesting/TestExplorerWindow.cs

```diff
@@ -13,16 +13,12 @@ private TestExplorerWindow()
 
         public TestExplorerWindow(TestExplorerViewModel viewModel) : this()
         {
-            _viewModel = viewModel;
-            wpfTestExplorerControl.DataContext = _viewModel;
-        }
-
-        private readonly TestExplorerViewModel _viewModel;
-        public TestExplorerViewModel ViewModel
-        {
-            get { return _viewModel; }
+            ViewModel = viewModel;
+            wpfTestExplorerControl.DataContext = ViewModel;
         }
+        public TestExplorerViewModel ViewModel { get; }
 
+        // FIXME bare ClassId... not good
```

Comment:
yeah. needs to be taken from `Rubberduck.Resources`

Reply:
We're currently only providing GUIDs and ProgIDs in Rubberduck.Resources.Registration. A solution here should also entail fixing this issue in other Dockable window hosts.

There already is a TODO comment for the same problem in `ToDoExplorerWindow`. For now I'd deem this out of scope for the current PR and would address in a dedicated follow-up issue (TBD) or the TODO/FIXME issue (#4076)